src/System.ComponentModel.Annotations/tests/FilterUIHintAttributeTests.cs

```diff
@@ -0,0 +1,88 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.ComponentModel.DataAnnotations.Tests
+{
+    public class FilterUIHintAttributeTests
+    {
+#pragma warning disable 618
```

Comment:
Can you add a comment next to this about why it's necessary?  Presumably it's to suppress the obsolete warnings for FilterUIHintAttribute usage?